tests/dashboard/dashboard-teams.spec.js

```diff
@@ -2050,9 +1873,7 @@ test.describe(() => {
       );
       await teamPage.clickOnRequestAccessButton();
       await teamPage.isRequestAccessButtonVisible(false);
-      await expect(teamPage.accessDialog).toHaveScreenshot(
-        'request-sent-dialog-image.png',
```

Comment:
Nice!
I think you may also the delete their snapshots from the repository (the same you did it with "rectangle-component-delete.png")

Reply:
Ok, I searched for usages on the file, but I may be wrong. I believe you.